classes/console.php

```diff
@@ -64,6 +64,29 @@ public static function help()
 
 	}
 
+	private const MAX_HISTORY = 99;
+	private $history = [];
```

Comment:
Class variables should be defined at the top, before the constructor, not in the middle of the code.
Also, never use "private", use "protected". All Fuel classes should be fully extendable.